ArchiSteamFarm/Storage/GlobalConfig.cs

```diff
@@ -504,6 +504,14 @@ public sealed class GlobalConfig {
 				return (null, null);
 			}
 
+			if (globalConfig.IPCPasswordFormat == ArchiCryptoHelper.EHashingMethod.PlainText) {
+				if (globalConfig.IsIPCPasswordSet && globalConfig.IPCPassword != null) {
```

Comment:
Same here, also merge those 3 `if` into one with three AND conditions, I hate nesting like that.